packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -211,6 +274,9 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
         Output: this.props.output?.s3Location ? {
           S3Uri: `s3://${this.props.output.s3Location.bucketName}/${this.props.output.s3Location.objectKey}`,
         } : undefined,
+        GuardrailIdentifier: this.props.guardrailConfiguration?.guardrailIdentifier,
+        GuardrailVersion: this.props.guardrailConfiguration?.guardrailVersion,
+        Trace: this.props.trace,
```

Comment:
```suggestion
        Trace: this.props.traceEnabled ? 'ENABLED' : 'DISABLED',
```

Reply:
Thank you. 
I'm concerned that with the proposed approach, if `traceEnabled` is undefined, `DISABLED` would be set in the template, potentially modifying existing templates. 
To account for `undefined` case, I'm considering the following approach. 
If you have any better suggestions, please let me know.

```ts
        Trace: this.props.traceEnabled === undefined
          ? undefined
          : this.props.traceEnabled
            ? 'ENABLED'
            : 'DISABLED',
```

Reply:
That's a valid concern and I agree with your proposed and improved solution.
Thanks for the feedback!